docs/2.getting-started.md

```diff
@@ -1,12 +1,153 @@
 # 2. Getting Started
 
-## Installation Options
+## Installation
 
-Currently, the Abilities API is available as a WordPress plugin. You can install it by cloning the repository into your `wp-content/plugins` directory.
+Until the Abilities API is merged into WordPress core, it must be installed before it can be used.
+
+### As a plugin
+
+The best and easiest way to try and use the Abilities API is to install it as a plugin by downloading the latest release from the [GitHub Releases page](https://github.com/WordPress/abilities-api/releases/latest).
```

Comment:
This requires us to cut a release (but so does composer). Can use the path to trunk in the interim, I just didn't want to rewrite something that's anyway needs to be rewritten by a competent docs writer